### PR TITLE
content/*: fix Node.js submission typos and formatting

### PR DESCRIPTION
**Pull request by odeke-em**
Addresses the stalled feedback from PR #261 for
semantic/typographic corrections. A bunch of the code
though still needs instructions on how to compile/generate
it, the right dependencies and pictures of traces.

### CODE REVIEW

**Comment by odeke-em** [comment]
@hvent90 when you have time, since you know Typescript, could you please help me send a follow-up PR that has code samples that compile correctly along with step by step instructions on how to get things to compile end-to-end?

File: content/guides/exporters/supported-exporters/Node.js/stackdriver-stats.md [hunk 1]
```diff
@@ -44,7 +44,7 @@ export GOOGLE_APPLICATION_CREDENTIALS=path/to/your/credential.json
 
 * Create the exporters in code
 
-#### Creating the exporter in code
+## Creating the exporter
```
Comment (songy23) [inline content/guides/exporters/supported-exporters/Node.js/stackdriver-stats.md]: Maybe just remove this tab? (This tab is removed in stackdriver-trace.md)

File: content/guides/exporters/supported-exporters/Node.js/stackdriver-trace.md [hunk 2]
```diff
@@ -1,41 +1,47 @@
...
 
+## Introduction
 Stackdriver Trace is a distributed tracing system that collects latency data from your applications and displays it in the Google Cloud Platform Console.
+
 You can track how requests propagate through your application and receive detailed near real-time performance insights.
 Stackdriver Trace automatically analyzes all of your application's traces to generate in-depth latency reports to surface performance degradations,
 and can capture traces from all of your VMs, containers, or Google App Engine projects.
 
-OpenCensus Node has support for this exporter available through package:
+Stackdriver Monitoring provides visibility into the performance, uptime, and overall health of cloud-powered applications.
```
Comment (songy23) [inline content/guides/exporters/supported-exporters/Node.js/stackdriver-trace.md]: I think this page is only about Stackdriver Tracing?

File: content/guides/exporters/supported-exporters/Node.js/stackdriver-stats.md (hunk 1 above)
Comment (odeke-em) [reply content/guides/exporters/supported-exporters/Node.js/stackdriver-stats.md]: Roger that, thanks!

File: content/guides/exporters/supported-exporters/Node.js/stackdriver-trace.md (hunk 2 above)
Comment (odeke-em) [reply content/guides/exporters/supported-exporters/Node.js/stackdriver-trace.md]: Oops, updating, thanks!

**Comment by odeke-em** [comment]
@songy23 I've updated with your suggestions, PTAL!

>hough I don't have too much context on the Node implementation.

Me neither. This PR is just a follow-up to unstall PR #261 as I mentioned in the commit message here :)

**Comment by odeke-em** [comment]
@songy23 I added a memo that will need the Node.js experts to address in https://github.com/census-instrumentation/opencensus-website/issues/284

**Comment by odeke-em** [comment]
Thank you @songy23!

**Pull request merged**